stanford_courses.info.yml

```diff
@@ -0,0 +1,55 @@
+name: 'Stanford Courses'
+type: module
+description: 'Courses content type'
+core_version_requirement: ^8.8 || ^9
+version: 8.2.7
+package: Stanford
+dependencies:
+  - datetime
+  - field_formatter_class
+  - field_group
+  - inline_entity_form
+  - layout_builder
+  - layout_discovery
+  - link
+  - menu_block
+  - menu_ui
+  - node
+  - options
+  - path
+  - taxonomy
+  - text
+  - ui_patterns_views
+  - user
+  - views_infinite_scroll
+  - rabbit_hole
+  - rh_node
+  - rh_taxonomy
+  - pathauto
+  - smart_trim
+  - taxonomy_menu
+  - config_pages_overrides
+  - config_pages
+  - scheduler
+  - shs
+  - jumpstart_ui
+  - layout_builder_restrictions
+  - layout_library
+  - ds
+  - migrate_plus
+  - migrate
+  - page_manager
+  - flat_taxonomy
+  - scheduler
+  - stanford_migrate
+  - ultimate_cron
+  - stanford_fields
+  - views_taxonomy_term_name_depth
+  - ui_patterns
+  - ui_patterns_layouts
+  - ui_patterns_library
+  - ui_patterns_views
+  - stanford_courses_importer
+components:
+  namespaces:
+    course: templates/components
```

Comment:
Let's namespace it something more specific like `stanford-courses`

Reply:
updated.